lib/hooks/useList.ts

```diff
@@ -14,7 +14,7 @@ interface PaginatedListContributorsResponse {
   meta: Meta;
 }
 
-const useFetchAllLists = (range = 30) => {
+const useFetchAllLists = (range = 30, fetch = true) => {
```

Comment:
Refactor (blocking): `fetch` is a function in JavaScript runtimes. Calling this boolean `fetch` could potentially confuse people, even though we're using TypeScript. I did a double take when I saw it thinking you were passing in the `fetch` function.